include/gamepad/controller.hpp

```diff
@@ -50,30 +178,79 @@ class Controller {
          * Updates the state of the gamepad (all joysticks and buttons), and also runs
          * any registered handlers.
          * @note This function should be called at the beginning of every loop iteration.
-         * @note Create a separate instance for each task.
+         *
+         * @b Example:
+         * @code {.cpp}
+         * while (true) {
+         *   Gamepad::master.update();
+         *   // do robot control stuff here...
+         *   pros::delay(25);
+         * }
+         * @endcode
+         *
          */
         void update();
         /**
          * Get the state of a button on the controller.
-         * @param button Which button's state you want.
+         * @param button Which button to return
+         *
+         * @b Example:
+         * @code {.cpp}
+         * if(Gamepad::master[DIGITAL_L1]) {
+         *   // do something here...
+         * }
+         * @endcode
+         *
          */
         const Button& operator[](pros::controller_digital_e_t button);
         /**
          * Get the value of a joystick axis on the controller.
-         * @param joystick Which joystick axis's value to return
+         * @param joystick Which joystick axis to return
```

Comment:
Padding before brief

```suggestion
         *
         * @param joystick Which joystick axis to return
```